planning/motion_velocity_planner/autoware_motion_velocity_run_out_module/src/run_out_module.cpp

```diff
@@ -155,6 +159,27 @@ void RunOutModule::publish_debug_trajectory(
   debug_trajectory_publisher_->publish(debug_trajectory);
 }
 
+void RunOutModule::add_planning_factors(
+  const std::vector<autoware_planning_msgs::msg::TrajectoryPoint> & trajectory,
+  const VelocityPlanningResult & result)
+{
+  if (trajectory.empty()) {
+    return;
+  }
+  geometry_msgs::msg::Pose p;
+  for (const auto & slowdown : result.slowdown_intervals) {
+    p.position = slowdown.from;
+    planning_factor_interface_->add(
+      trajectory, trajectory.front().pose, p, PlanningFactor::SLOW_DOWN, SafetyFactorArray{}, true,
+      slowdown.velocity);
+  }
+  for (const auto & stop : result.stop_points) {
+    p.position = stop;
+    planning_factor_interface_->add(
+      trajectory, trajectory.front().pose, p, PlanningFactor::STOP, SafetyFactorArray{}, true, 0.0);
```

Comment:
@maxime-clem I would also like the SafetyFactor information to be stored correctly. In the case of this module, it is assumed that the information pertains to an object that may potentially dart out and triggers Ego to decelerate or stop. It's fine to handle this in a separate PR, but I would appreciate it if you could take care of it.